src/compression.rs

```diff
@@ -49,7 +56,7 @@ pub enum CompressionMethod {
 /// All compression methods defined for the ZIP format
 impl CompressionMethod {
     pub const STORE: Self = CompressionMethod::Stored;
-    pub const SHRINK: Self = CompressionMethod::Unsupported(1);
+    pub const SHRINK: Self = CompressionMethod::Shrink;
     pub const REDUCE_1: Self = CompressionMethod::Unsupported(2);
     pub const REDUCE_2: Self = CompressionMethod::Unsupported(3);
     pub const REDUCE_3: Self = CompressionMethod::Unsupported(4);
```

Comment:
Change these to `CompressionMethod::Reduce`.

Reply:
+1.